Base/QTCore/qSlicerScriptedFileWriter.h

```diff
@@ -71,6 +74,19 @@ class Q_SLICER_BASE_QTCORE_EXPORT qSlicerScriptedFileWriter
   /// \sa qSlicerFileWriter::write()
   bool write(const qSlicerIO::IOProperties& properties) override;
 
+  /// Added so node writers can report back written nodes
+  /// \sa qSlicerFileWriter::writtenNodex()
+  void addWrittenNode(const QString& writtenNode);
+
+  /// Reimplemented to propagate to python methods
+  /// \sa qSlicerFileWriter::writtenNodes()
+  QStringList writtenNodes()const override {
+    return Superclass::writtenNodes();
```

Comment:
Did you do this because you had to expose the method on public interface and did not want to modify method visibility in the base class? If yes, then please add a comment that explains it.

Reply:
Including this no-op override of `writtenNodes` is not required, but to me it makes this code more readable and self contained.  I'll add more comments.